grid2op/tests/test_alert.py

```diff
@@ -13,45 +13,1234 @@
+                else : 
+                    assert reward == 0
+
+                if done: 
+                    break
+
+
+# return 2
+    def test_assistant_reward_value_blackout_attack_raise_good_alert(self) -> None :
+        kwargs_opponent = dict(lines_attacked=[ATTACKED_LINE], 
+                                   duration=3, 
+                                   steps_attack=[2])
+        with make(self.env_nm, test=True, difficulty="1", 
+                            opponent_attack_cooldown=0, 
+                            opponent_attack_duration=99999, 
+                            opponent_budget_per_ts=1000, 
+                            opponent_init_budget=10000., 
+                            opponent_action_class=PlayableAction, 
+                            opponent_class=TestOpponent, 
+                            kwargs_opponent=kwargs_opponent
+            ) as env : 
+            new_param = Parameters()
+            new_param.MAX_LINE_STATUS_CHANGED = 10
+
+            env.change_parameters(new_param)
+            env.seed(0)
+            env.reset()
+            for i in range(env.max_episode_duration()):
+                attackable_line_id = 0
+                act = self.do_nothing
+                if i == 3 : 
+                    act = self.blackout_action
+                elif i == 1:
+                    act = self.env.action_space({"raise_alert": [attackable_line_id]})
+                obs, reward, done, info = env.step(act)
+                if i == 2 : 
+                    assert env._oppSpace.last_attack is not None
+                
+                if i == 3: 
+                    assert reward == 2
+                else : 
+                    assert reward == 0
+
+                if done: 
+                    break
+
+
+# return -10
+    def test_assistant_reward_value_blackout_attack_raise_alert_just_before_blackout(self) -> None :
+        """
+        When 1 line is attacked at step 2 and we raise 1 alert  too late
+        we expect a reward of -10 at step 3 
+        """
+        kwargs_opponent = dict(lines_attacked=[ATTACKED_LINE], 
+                                   duration=3, 
+                                   steps_attack=[2])
+        with make(self.env_nm, test=True, difficulty="1", 
+                            opponent_attack_cooldown=0, 
+                            opponent_attack_duration=99999, 
+                            opponent_budget_per_ts=1000, 
+                            opponent_init_budget=10000., 
+                            opponent_action_class=PlayableAction, 
+                            opponent_class=TestOpponent, 
+                            kwargs_opponent=kwargs_opponent
+            ) as env : 
+            new_param = Parameters()
+            new_param.MAX_LINE_STATUS_CHANGED = 10
+
+            env.change_parameters(new_param)
+            env.seed(0)
+            env.reset()
+            for i in range(env.max_episode_duration()):
```

Comment:
can you remove this "for" loop and make the thing sequential (you only need to do 3 actions, 3 steps and 3 checks, it's clearer without the for loop I think)